modules/videoio/include/opencv2/videoio.hpp

```diff
@@ -77,7 +81,8 @@ To enable/disable APIs, you have to:
      (e.g. <tt>-DWITH_MSMF=ON -DWITH_VFW=ON ... </tt>) or checking related switch in cmake-gui
   2. rebuild OpenCV itself
 */
-enum { CAP_ANY          = 0,            //!< Auto detect
+enum VideoCaptureApiID {
```

Comment:
> and use plural forms

I believe it is useless and looks strange in general.
But it is already [happened](https://github.com/opencv/opencv/commit/6aa7a86ae992d3aacaf898c2d7c791d73311d2dd#diff-82807d0ed5d1eae81b6cb4467af33e92R178)...